packages/effects-core/src/components/loop-subd-component.ts

```
@@ -0,0 +1,337 @@
+  private subdivisionLevel = 1; // 细分级别
+  private wireframe = false; // 是否使用线框模式
+
+  private effectComponent: EffectComponent;
+
+  constructor (engine: Engine) {
+    super(engine);
+  }
+
+  override onStart (): void {
+    // 获取EffectComponent
+    this.effectComponent = this.item.getComponent(EffectComponent);
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)